### DISCUSSION TITLE
Option to backup only in a specific backend

### DISCUSSION THREAD

**Discussion by ioparaskev** [cupcakearmy/autorestic#80] [Q&A] [answered]
Is there an option to backup only to a specific backend instead of all? I failed to find a related option

**Comment by cupcakearmy** [comment, marked as answer]
Not right now no.
But i can see how that can be useful, maybe something like an `--only` flag for backups?

**Reply by fdw** [reply]
That would certainly help. I also like [crestic](https://github.com/nils-werner/crestic)'s naming with `<location>@<backend>`, f.e. `home@hdd`. If something like that would be possible, that'd be awesome! 🙂

**Reply by cupcakearmy** [reply]
Addressed in https://github.com/cupcakearmy/autorestic/issues/116

**Reply by cupcakearmy** [reply]
Is implemented now
